src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -2836,7 +2850,7 @@ public String getDescription()
          *
          * @return {@link java.util.concurrent.CompletableFuture} - Type: {@link java.io.InputStream}
          */
-        @Nonnull
+        @Nonnull //TODO remove
         public CompletableFuture<InputStream> retrieveInputStream() // it is expected that the response is closed by the callback!
```

Comment:
Lets mark all these as deprecated for now with a `ReplaceWith` annotation to show `getProxy`